daprdocs/content/en/operations/components/setup-secret-store/supported-secret-stores/azure-keyvault.md

```diff
@@ -40,13 +40,25 @@ The above example uses secrets as plain strings. It is recommended to use a loca
 
 ## Spec metadata fields
 
+### Self-Hosted
+
 | Field              | Required | Details                                                                 | Example                  |
 |--------------------|:--------:|-------------------------------------------------------------------------|--------------------------|
 | vaultName          | Y        | The name of the Azure Key Vault                                         | `"mykeyvault"`           |
 | spnTenantId        | Y        | Service Principal Tenant Id                                        | `"spnTenantId"`          |
 | spnClientId        | Y        | Service Principal App Id                                           | `"spnAppId"`             |
 | spnCertificateFile | Y        | PFX certificate file path. <br></br> For Windows the `[pfx_certificate_file_fully_qualified_local_path]` value must use escaped backslashes, i.e. double backslashes. For example `"C:\\folder1\\folder2\\certfile.pfx"`. <br></br> For Linux you can use single slashes. For example `"/folder1/folder2/certfile.pfx"`.  <br></br> See [configure the component](#configure-the-component) for more details | `"C:\\folder1\\folder2\\certfile.pfx"`, `"/folder1/folder2/certfile.pfx"`        |
 
+
+### Kubernetes
+
+| Field              | Required | Details                                                                 | Example                  |
+|--------------------|:--------:|-------------------------------------------------------------------------|--------------------------|
+| vaultName          | Y        | The name of the Azure Key Vault                                         | `"mykeyvault"`           |
+| spnTenantId        | Y        | Service Principal Tenant Id                                        | `"spnTenantId"`          |
+| spnClientId        | Y        | Service Principal App Id                                           | `"spnAppId"`             |
+| spnCertificate     | Y        | Store the certificate for the service principal into the Kubernetes Secret Store.<br></br> See [configure the component](#configure-the-component) for more details | kubectl create secret generic [name] --from-file=[fully_qualified_local_path]        |
```

Comment:
Love the instructions on how to create and store the secret, but the command to get the secret in the store should be in the Description. There should also be instructions here on how to reference the certificate secret once it's in the store. The Example should be what to place in the spec yaml file instead of how to get it there.

Reply:
I agree the document could be more complete with the steps on how to store the file as a secret in K8s. I will work on that this week.

Reply:
This was already in the table 

```
kubectl create secret generic [name] –from-file=[fully_qualified_local_path]
```
Do you want more than that?